Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/MainPage.xaml.cs

```diff
@@ -1,4 +1,7 @@
-﻿// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
```

Comment:
I'm missing the context for this additional line here but my guess it's fine. Leaving this comment to make sure other see it while they look at this PR.

Reply:
It was already in the code base, i think it's fine to leave as is.